GOVERNANCE.md

```diff
@@ -37,7 +37,19 @@
 
 ## When does a maintainer lose maintainer status
 
-If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they should volunteer to be removed from the [OWNERS list](github.com/cri-o/cri-o/tree/master/OWNERS). In extreme cases this can also occur by a vote of the maintainers per the voting process below.
+If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they may volunteer to be removed from the [OWNERS list](github.com/cri-o/cri-o/tree/master/OWNERS).
+
+In extreme cases this can also occur by a vote of the maintainers per the voting process below.
+
+They may also opt to be come an [emeritus maintainer](#emeritus-maintainer)
+
+# Emeritus Maintainer
+
+An emeritus maintainer is a maintainer who used to spend a significant amount of time on the project, but now has refocused their efforts.
+
+The CRI-O community has the emeritus maintainer tier to recognize the impact and input early maintainers have had on the project, while also being realistic about their current committment level.
+
+There is no formal process for becoming an emeritus maintainer: a maintainer simply slows down on their contributions but does not remove themselves from the [OWNERS file](OWNERS.md).
```

Comment:
@haircommander, I really like the "Emeritus" idea, GNOME dos something quite similar.

If I may make a suggestion, I'd prefer having the Emeritus Maintainers listed as so. By doing this new contributors will know who they can ping, who they can ask for help, etc.  With no clear distinction between both may be a slightly confusing for new contributors (I must admit it was slightly confusing for me when I first arrived here a few months ago).